doc/modules/manifold.rst

```diff
@@ -305,7 +305,7 @@ The overall complexity of standard HLLE is
 Spectral Embedding
 ====================
 
-Spectral Embedding (also known as Laplacian Eigenmaps) is one method
+Spectral Embedding (specifically, Laplacian Eigenmaps) is one method
```

Comment:
Perhaps "Spectral Embedding is an approach to calculating a non-linear embedding. Scikit-learn implements Laplacian Eigenmaps, which finds a low dimensional representation ..."

but this is not an area of my expertise.